CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 cmake_minimum_required(VERSION 3.7)
-cmake_policy(SET CMP0048 NEW)
+cmake_policy(SET CMP0091 NEW) # Support MSVC_RUNTIME_LIBRARY
```

Comment:
This policy is introduced in CMake 3.15 so my vote for a bump from 3.7 to 3.15 above or for a version check guard.

Reply:
Besides this feature on Windows we don't use anything from CMake past 3.7.

There is a version check in ``ConfigureWindows.cmake`` (and a noisy warning) already that they should update to CMake 3.15+ on Windows. Afaik CMake does not warn about unknown policies so an if-endif here would add 2 more unnecessary lines.

Reply:
I don't agree but your call :smiley:

Reply:
There is really a if check required around the policy... otherwise this throws an error xD